doc/tutorials/content/sources/function_filter/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8 FATAL_ERROR)
+
+project(function_filter)
+
+find_package(PCL 1.11.1 REQUIRED)
```

Comment:
It would require 1.12 due to the additions to `functor_filter.h` when released?

Reply:
Would 1.11.1.99 work?

Reply:
I would think so - we just need to change it before release 👍

Reply:
```suggestion
find_package(PCL 1.11.1.99 REQUIRED)
```